src/udev/udev-config.c

```diff
@@ -112,6 +112,20 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
 
                 return 0;
 
+        } else if (proc_cmdline_key_streq(key, "udev.trace")) {
```

Comment:
plain streq will do?

Reply:
i'd probably always do proc_cmdline_key_streq() without thinking about it too much, just to keep things uniform. not too religious about this myself though

(but yes, an streq() would do here too)

Reply:
Yeah, streq is sufficient, but for consistency with others.